packages/did-core-test-server/suites/did-spec/did-metadata-structure.js

```diff
@@ -0,0 +1,61 @@
+const didMetadataStructureTest_r = (didDocumentMetadata) => {
+    describe("didDocumentMetadata", () => {
```

Comment:
if you can prefix this with the section of the spec that will help us make sure we are covering everything

Reply:
It's a function called from main `describe` section. For the source-code-wise, prefix might be useful, but if we ran test verbosely, the section number appears multiple times. May let me consider what is the best solution in future commits.